package book.ch14;

public class UserExceptionTest {
	public void test() throws UserException {
		System.out.println("test 호출 성공");
	}

	public static void main(String[] args) {
		// TODO Auto-generated method stub

	}

}
